tests/configs/wall_loss_config/my_config.json

```diff
@@ -22,7 +22,8 @@
         }
     },
     "initial conditions": {
-        "initial_conditions.csv": {}
+        "PHOT.LOSS_SOA1 wall loss.s-1": 0.01,
+        "PHOT.LOSS_SOA2 wall loss.s-1": 0.05
```

Comment:
Why isn't this read in from the initial conditions file?

Reply:
This was how the box model was generating the config, however I can make it generate a file instead.

Reply:
I changed the config to use the initial_conditions.csv to store and read these configs.